src/Dfc.CourseDirectory.WebV2/Features/Venues/AddVenue/CheckAndPublish.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@model Dfc.CourseDirectory.WebV2.Features.Venues.AddVenue.CheckAndPublish.ViewModel
+@{
+    Layout = "_V2LayoutProviderContext";
+    ViewContext.SetLayoutData(title: "Add location");
+}
+
+@if (Model.NewAddressIsOutsideOfEngland)
+{
+    <pttcd-notification title="Non England address" data-testid="outsideofengland-notification">
+        You are adding an address that is outside of England. Your course for this address will not appear in any search results.
```

Comment:
Is that really the copy that's been supplied for this error message?

"Your course for this address will not appear in any search results." is poorly worded, particularly the use of "course".

Reply:
Copy is taken from the prototype

Reply:
Can we raise this with UX/PO please?